examples/src/pages/Clipping.tsx

```diff
@@ -22,6 +19,7 @@ export function Clipping() {
 
   useEffect(() => {
     const gui = new dat.GUI();
+    let revealManager: reveal.RevealManager<unknown>;
```

Comment:
It's not really unknown, we either use modelId + revisionId or modelUrl, don't we?

I would personally like to have some sane defaults for that template identifier. For most cases, it probably would be modelId + revisionId.

Reply:
This is more complicated than just changing it. I will setup a proposal for our examples later which will change this behavior.